template.yaml

```
@@ -118,6 +205,83 @@ Resources:
+            ProjectionType: ALL
+      # Enable point-in-time recovery for continuous backup
+      PointInTimeRecoverySpecification:
+        PointInTimeRecoveryEnabled: true
+
+  # SNS Topic for Lockbox Events
+  LockboxEventsTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: lockbox-events
+      DisplayName: Lockbox Events
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)